shell.js

```diff
@@ -18,6 +18,21 @@ var common = require('./src/common');
 //@ page](https://github.com/shelljs/shelljs/wiki).
 //@
 
+// Boilerplate
+// -----------
+// Copy the code block below here & replace variables with appropiate values
+// ```
+// //@include ./src/fileName
+// var functionName = require('./src/fileName');
+// exports.nameOfCommand = common.wrap(nameOfCommand, functionName, {idx: firstIndexToExpand});
+// ```
+//
+// The //@include includes the docs for that command
+//
+// firstIndexToExpand should usually be 1 (so, put {idx: 1})
+// Increase this value if the command takes arguments that shouldn't be expanded
+// with wildcards, such as with the regexes for sed & grep
+
```

Comment:
This is just an explanation of the boilerplate code. In particular, this explains what `{idx: 1}` is all about (which is not very intuitive otherwise).